content/en/faq/metrics-and-logs/telemetry-v1-vs-v2.md

```diff
@@ -22,6 +22,9 @@ v2 which are listed below:
   missing peer attributes like workload name, namespace and labels.
   However, if both peers have proxies injected all the labels mentioned
   [here](/docs/reference/config/metrics/) are available in the generated metrics.
+  At Istio 1.8 release, a solution was added to distribute server workload metadata
+  to client sidecar, which makes client side metric have server workload metadata labels
+  filled even when the server workload is out of the mesh.
```

Comment:
The way the section reads now is a bit confusing. If starting from 1.8, it now has workload labels for server workloads that are not part of the mesh, then why not just remove this whole section? It seems strange to say there is a problem and then say the solution is there since an older Istio release.

Reply:
Stats reporting both at client proxy and server proxy. The 1.8 solution only resolves missing server metadata at client side. Stats reporting at server side still lacks metadata is the client is outside the mesh.